types/types_test.go

```diff
@@ -35,3 +36,27 @@ func TestHeaders_ShouldReturnTrueWhenHasSecureHeadersDefined(t *testing.T) {
 
 	assert.True(t, headers.HasSecureHeadersDefined())
 }
+
+func TestHeaderRedactions_OneItemShouldBeOneItem(t *testing.T) {
+	redactions := HeaderRedactions{}
+	redactions.Set("Authorization")
+	assert.True(t, len(redactions) == 1)
+}
+
+func TestHeaderRedactions_TwoItemsShouldBeTwoItems(t *testing.T) {
+	redactions := HeaderRedactions{}
+	redactions.Set("Authorization,User-Agent")
+	assert.True(t, len(redactions) == 2)
+}
+
+func TestHeaderRedactions_EmptyItemsShouldBeSkipped(t *testing.T) {
+	redactions := HeaderRedactions{}
+	redactions.Set(",Authorization,")
+	assert.True(t, len(redactions) == 1)
+}
+
+func TestHeaderRedactions_WhitespaceShouldBeStripped(t *testing.T) {
+	redactions := HeaderRedactions{}
+	redactions.Set(" Authorization  ")
+	assert.False(t, strings.Contains(redactions[0], " "))
+}
```

Comment:
I think the first three tests could be combined into a single table-driven test using (parallelized) subtests. And if we were willing to test the content as well (which I believe would be a worthwhile addition), we could even cover all four tests.

WDYT?